src/appMain/sdl_preloaded_pt.json

```diff
@@ -854,8 +854,7 @@
                         "hmi_levels": [
                             "FULL",
                             "LIMITED",
-                            "BACKGROUND",
-                            "NONE"
+                            "BACKGROUND"
```

Comment:
Changing default permissions could likely break someones integration. Is this really necessary?

Reply:
Not strictly, this change just made the most sense security-wise after looking into all of the cases for this issue. It doesn't really make too much sense to allow an app to publish an active service before the actual app is activated at least once.

As for breaking integrations, I really don't think that this file should be considered definitive in any way, policies are heavily configurable for a reason. If an app assumes that every integration has the same general policy configurations, it's going to run into issues anyway.

Reply:
> If an app assumes that every integration has the same general policy configurations, it's going to run into issues anyway.

Excellent point